blog/_posts/2017-10-3-RGSoC- the grand finale.md

```diff
@@ -0,0 +1,99 @@
+---
+title: RGSoC- the Grand Finale.
+layout: post
+created_at: Tue October 3 2017
+permalink: blog/2017-10-3-RGSoC- the Grand Finale.
```

Comment:
File name and permalink should match eachother. Permalink dont need to have file format.